Scala/Scala.sublime-syntax

```diff
@@ -610,6 +708,43 @@ contexts:
     - match: '(?=\S)'
       pop: true
 
+  function-parameter-list-contents:
```

Comment:
How about adding meta scopes? Those would be required to reliably support syntax based folding.

```suggestion
  function-parameter-list-contents:
    - meta_scope: meta.group.scala
```

Maybe also `meta.function.parameters` if it is feasable to scope function definitions.

Reply:
Function definitions are going to be really tricky to scope. Same with class definitions. I'd really *like* to though.